js/lib/bandstructure.js

```diff
@@ -332,6 +332,12 @@ BandPlot.prototype.initDosChart = function (orientation = 'vertical') {
                 legend: {
                     display: this.showLegend,
                     position: 'right',
+                    labels: {
+                        filter: function(item, chart) {
+                        // Logic to remove a particular legend item goes here
+                            return !item.text.includes('test');
```

Comment:
Is this leftover for test purpose?

Reply:
I will change to a more reasonable label.